tests/BuildScriptGeneratorCli.Tests/BuildpackDetectCommandTest.cs

```diff
@@ -100,6 +100,7 @@ private static IServiceProvider GetServiceProvider(BuildpackDetectCommand cmd)
                 {
                     svcs.Replace(ServiceDescriptor.Singleton(typeof(IEnvironment), env));
                     svcs.AddSingleton<INodeVersionProvider, TestNodeVersionProvider>();
+                    svcs.AddSingleton<IPhpVersionProvider, TestPhpVersionProvider>();
```

Comment:
why?>

Reply:
nit: Can we change the filename BuildpackDetectCommandTest to something else? I was initially confused .. I thought we adding php in buildpack .. so I made that "why" comment

Reply:
I added this because this was failing a test. In this PR I modified the logic as per the spec.